Editing notes using the github.dev editor.md

```diff
@@ -0,0 +1,63 @@
+
+## Opening the github.dev editor
+
+To open the editor you'll usually have two options: 
+1. On the Hub's [publish site](https://publish.obsidian.md/hub/00+-+Start+here), each note has an "Edit in GitHub" button in its footer.
+2. When you are in the Hub's [GitHub repository](https://github.com/obsidian-community/obsidian-hub), pressing <kbd>.</kbd> will take you to the editor. 
+
+Note that you will need a GitHub account to make edits, and if you aren't logged in to it, launching the github.dev editor will prompt you for your GitHub login details.
+
+## Making changes to the notes
+
+1. To make changes, you can select files from the file explorer (1), and make edits to the note (2) as you would on any text editor or paste any changes made in Obsidian to existing notes. To add new files, you can also drag from your system's file explorer to the file explorer in the editor (1).
+	![[github.dev editor.png]]
+2. Once you are ready to submit your changes, switch to the version control tab. You'll see the names of the files you've modified under the "Changes" section:
+    ![[github.dev editor version control.png|300]]
+3. Add the files you want to submit by pressing the <kbd>+</kbd> sign (2), and write a message describing the changes you've made(1): 
+	![[github.dev editor stage changes.png|300]]
+4. To submit your files, you have to press the <kbd>✓</kbd> button at the top. 
+	The first time you make an edit to the hub, you'll get a message explaining that GitHub will create a copy of the repository on your account. This is called a **Fork**.
+	![[github.dev editor commit changes.png]]
+5. Next, it will ask you to give a name to the *branch* where you want to save your changes. You can choose a new name for it, or just press enter. ^8cdfd9
+	![[github.dev editor create branch.png]] 
+6. GitHub will let you know it will switch now to the copy of the repository in your account. And once it reloads you'll see that your changes have been saved.
+	![[github.dev switch to fork.png|300]]
+	![[github.dev fork view.png]]
+
+At this stage, your edits are in your fork of the Hub.
+
+## Submitting changes to the Hub
+
+Now you need to submit the changes in Fork to the original Hub repository.
+
+Using git terminology, this is called creating a **Pull Request**.
+
+1. Clicking on [this](https://github.com/obsidian-community/obsidian-hub/compare) link is equivalent to going to the Hub repository, clicking on the Pull Requests tab and choosing New. Click on compare across forks, and choose yours on head repository. For the branch, switch to the one you chose on [[#^8cdfd9|step 6]] on the previous section:
+	![[github compare forks.png]]
+2. Once you have selected your fork and branch, you can choose <kbd>Create pull request</kbd>
+	![[github create pull request.png]] 
+3. Fill out the template and when you are done, press on <kbd>Create pull request</kbd>.  
```

Comment:
for the less git experienced, it will be confusing why you have to describe your changes again, after you already described your changes above in the commit message 🤔

Reply:
It already adds the title from the commit message. If a second or third line was used (I think not possible with this interface), then that would show up there.
And the contributor didn't describe what changes they made before, did they?